### DISCUSSION TITLE
Problema installazione HACS

### DISCUSSION THREAD

**Discussion by sayro85** [indomus/forum#2757] [HUB: Home Assistant]
Salve a tutti. Oggi durante l'installazione di HACS mi ha restituito il seguente errore
![Screenshot_20231207-234612](https://github.com/indomus/forum/assets/67622536/a5cbd585-d06c-4485-9f4c-dc80bcd4d73f)

Bisogna solo aspettare che facciano qualche aggiornamento?

**Comment by snakuzzo** [comment]
sì

**Comment by indomus** [comment]
> Salve a tutti. Oggi durante l'installazione di HACS mi ha restituito il seguente errore ![Screenshot_20231207-234612](https://private-user-images.githubusercontent.com/67622536/288912686-a5cbd585-d06c-4485-9f4c-dc80bcd4d73f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNzQ1OTksIm5iZiI6MTcwMjM3NDI5OSwicGF0aCI6Ii82NzYyMjUzNi8yODg5MTI2ODYtYTVjYmQ1ODUtZDA2Yy00NDg1LTlmNGMtZGM4MGJjZDRkNzNmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjEyVDA5NDQ1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTgzYTY2ZmY4ZjRiMDhlMWIzYmFhNTYwNmI1OTc5MTE3NzNjZDZmNjRkOGM2ZjU3NzgzNzgwYWVmZGU5Y2QxOTcmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.g1-6wAmxiE2-6oK-4YMDWlpIX3c0ApJFKUHq_yfNvGE)
> 
> Bisogna solo aspettare che facciano qualche aggiornamento?

https://github.com/hacs/integration/issues/3384#issuecomment-1847956073
